app/models/components/course/videos_ability_component.rb

```diff
@@ -61,6 +63,14 @@ def allow_student_update_own_video_submission
     can :update, Course::Video::Submission, video_submission_own_course_user_hash
   end
 
+  def allow_student_create_video_watch_interval
+    can :create, Course::Video::Session, submission: video_submission_own_course_user_hash
```

Comment:
Actually if the `Course::Video::Submission::SessionsController` uses the `update` endpoint, I don't think this is required. You can try omitting this permission and see if students are allowed to update their own video watch interval?

Reply:
Ah, I actually forgot to remove this. I'll take it out.

I intend for sessions to be automatically created when we navigate to the edit submission page (there's no reason not to).